packages/ui/src/DisplayModeToggle/Button/index.css

```diff
@@ -28,7 +28,7 @@
 .ory-controls-mode-toggle-button-description {
   font-family: Roboto, sans-serif;
   font-size: 16px;
-  margin-top: 24px;
+  margin-top: 18px;
```

Comment:
Reason for this change?

Reply:
I am pretty sure it was breaking something but couldn't remember what, give me a sec.

Reply:
Oh yes, I see it now, if you hover over the mode buttons. A little tool tip pops up. And it wasn't centered vertically, was driving me nuts :D So it's basically a fix. Unrelated to the plugin though, sorry about that. Unless it's intentional of course, then I can revert it.

Reply:
Ok, let's keep it!